argparse.go

```diff
@@ -18,15 +18,16 @@ var disableHelp = false
 // Command MUST NOT ever be created manually. Instead one should call NewCommand method of Parser or Command,
 // which will setup appropriate fields and call methods that have to be called when creating new command.
 type Command struct {
-	name        string
-	description string
-	args        []*arg
-	commands    []*Command
-	parsed      bool
-	happened    bool
-	parent      *Command
-	HelpFunc    func(c *Command, msg interface{}) string
-	exitOnHelp  bool
+	name               string
+	description        string
+	args               []*arg
+	commands           []*Command
+	parsed             bool
+	happened           bool
+	parent             *Command
+	HelpFunc           func(c *Command, msg interface{}) string
+	exitOnHelp         bool
+	DescriptionPadding interface{}
```

Comment:
I don't think this should be added here. It is unnecessary to have description padding on the command. Command type has nothing to do with formatting text. So it should not be here.